src/status_im/ui/screens/chats_list/subs.cljs

```diff
@@ -12,6 +12,7 @@
   :<- [:get :chats]
   :<- [:get-in [:toolbar-search :text]]
   (fn [[chats search-text]]
-    (if search-text
-      (filter #(search-filter search-text (second %)) chats)
-      chats)))
+    (let [chats' (if search-text
```

Comment:
Can we use a better var name? Or extract this part to a separate function?

Reply:
@jeluard How about `unordered-chats`? I initially did a separate fn / subscription, but this is the only place it is used so inline makes more sense  to me. Happy to change it if you prefer a separate function.

Reply:
unordered-chats is a good name :+1: